angr/state_plugins/inspect.py

```diff
@@ -158,7 +158,12 @@ def check(self, state, when):
                 l.debug("...... both None, True")
                 c_ok = True
             elif current_expr is not None and needed is not None:
-                if state.solver.solution(current_expr, needed):
+                solver_type_ce = isinstance(current_expr, claripy.ast.Base) or isinstance(current_expr, angr.state_plugins.sim_action_object.SimActionObject)
+                solver_type_n = isinstance(needed, claripy.ast.Base) or isinstance(needed, angr.state_plugins.sim_action_object.SimActionObject)
+                if current_expr == needed:
+                    l.debug("...... is identical!")
+                    c_ok = Truee
```

Comment:
Typo?

Reply:
Yep.  Should be fixed by recent commits.  Many thanks for the catch :)